### PR TITLE
worked on firebase function

### PR DESCRIPTION
**Pull request by mukeshsin**

### CODE REVIEW

File: config/database.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const Sequelize = require("sequelize");
+
+const db = new Sequelize("fbauth", "root", "singh@2023", {
```
Comment (meanstack9716) [inline config/database.js]: @mukeshsin why do you need a MySQL database.
You should use fireabse database

File: controllers/auth.controllers.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const admin = require("firebase-admin");
+
+// Initialize the Firebase Admin SDK
+const serviceAccount = require("../nodeassign-firebase-adminsdk-7rp5j-0a456fe229.json");
+admin.initializeApp({
```
Comment (meanstack9716) [inline controllers/auth.controllers.js]: move this initialization code in fireabse-config.js file @mukeshsin

File: models/auth.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+//import sequelize
+const Sequelize = require("sequelize");
+const db = require("../config/database.js");
+
+const User = db.define(
+  "users",
```
Comment (meanstack9716) [inline models/auth.js]: not needed remove it @mukeshsin

**Review by meanstack9716 — CHANGES_REQUESTED** [review]

File: controllers/auth.controllers.js (hunk 2 above)
Comment (mukeshsin) [reply controllers/auth.controllers.js]: ok sir @meanstack9716

File: config/config.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const Sequelize = require("sequelize");
```
Comment (meanstack9716) [inline config/config.js]: remove Sequelize @mukeshsin

**Review by meanstack9716 — CHANGES_REQUESTED** [review]

File: config/config.js (hunk 4 above)
Comment (mukeshsin) [reply config/config.js]: ok sir @meanstack9716  done

Comment (mukeshsin) [reply config/config.js]: can i merge it start next
@meanstack9716

**Pull request merged**